smoke.json

```diff
@@ -1,4 +1,34 @@
 {
+  "__inputs": [
+    {
+      "name": "DS_METRICS",
```

Comment:
@justelad this is messing up the dashboards when Grafana loads this JSON. Did you try to actually use this prior to the PR?